netdata.spec.in

```diff
@@ -206,6 +218,12 @@ BuildRequires: cups-devel >= 1.7
 %endif
 # end - cups plugin dependencies
 
+# go.d.plugin dependencies
+%if %{with goplugin}
+BuildRequires:  golang
```

Comment:
```suggestion
%if 0%{?suse_version}
BuildRequires: go
%else
BuildRequires:  golang
%endif
```

Needed so that these changes work on openSUSE.

Reply:
Thanks